packages/api-gateway/src/drivers/http/routes/user/schema.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
Can you add `tags` and `description` to your schema?

As part of #118 I added Swagger-ui to the api-gateway, and swagger uses this properties to group and describe the endpoints.

You can see a full example here: https://github.com/Platzi-Master-C9/booking-services/tree/master/packages/api-gateway#add-a-new-endpoint

```suggestion
module.exports = {
        description: 'Creates a new user in database',
        tags: ['Users'],
```

Reply:
I didn't know, I will apply it